apps/website/src/content/docs/tooltip.mdx

```diff
@@ -1,36 +1,73 @@
 ---
 title: Tooltip
-description: Tooltips display informative text when users hover over, focus on, or tap an element.
-propsPath: '@itwin/itwinui-react/esm/core/Tooltip/Tooltip.d.ts'
+description: Tooltips display informative text when users hover, focus, or tap an interactive element.
 thumbnail: Tooltip
+related:
+  - popover
+  - expandableblock
 ---
 
 <p>{frontmatter.description}</p>
 
-<LiveExample src='Tooltip.main.tsx' truncate={false}>
-  <AllExamples.TooltipMainExample client:load />
-</LiveExample>
-
-A tooltip is a small container that typically appears above the element you are hovering with your cursor with a short, helpful text message.
+Make sure to read the [Accessibility](#accessibility) section for best practices before using tooltips.
 
 ## Usage
 
-You should use up to 256 characters. Content other than text such as pictures, tables, etc., are not recommended. Mobile devices do not have a hover state. Thus, be cautious when using tooltips in desktop apps that have a mobile counterpart, since there is no determined way of replacing tooltips on mobile devices.
+Wrap `Tooltip` around the element you want to use as the "trigger", and pass the tooltip text into the `content` prop.
 
-Examples of the many ways where tooltips can be used: [AvatarGroup](avatar#overflow), [Stepper](stepper#tooltip), and [Slider](slider#tooltips).
+<LiveExample src='Tooltip.main.tsx' truncate={false}>
+  <AllExamples.TooltipMainExample client:load />
+</LiveExample>
 
-It is not recommended to use tooltips to display critical information, or as a means to navigate an interface. They are necessary only when supplemental information is required. Functionality of the product should NOT be dependent on the user reading tooltips.
+For everything to work correctly, the trigger element must:
 
-Tooltips are enabled by default but can be turned off.
+- be an interactive element, for example, a `<button>`
+- forward its ref
+- delegate (spread) any arbitrary props
 
-### Placement
+## Positioning
 
-Tooltips can also be customized such as changing the placement.
+`Tooltip` handles positioning using an external library called [Floating UI](floating-ui.com/). To control which side the tooltip should be placed relative to its trigger, use the `placement` prop (which defaults to `"top"`). If not enough space is available, then it will automatically flip to the opposite side.
```

Comment:
The Floating UI link is broken. Also fluency (nit).

```suggestion
`Tooltip` handles positioning using an external library called [Floating UI](https://floating-ui.com/). To control which side the tooltip should be placed relative to its trigger, use the `placement` prop (which defaults to `"top"`). If enough space is not available, then it will automatically flip to the opposite side.
```